tests/acm.rs

```diff
@@ -0,0 +1,25 @@
+#![cfg(feature = "acm")]
+
+extern crate rusoto;
+
+use rusoto::acm::{AcmClient, ListCertificatesRequest};
+use rusoto::{DefaultCredentialsProvider, Region};
+
+#[test]
+fn should_list_certificates() {
+    let credentials = DefaultCredentialsProvider::new().unwrap();
+    let client = AcmClient::new(credentials, Region::UsEast1);
+
+    let request = ListCertificatesRequest::default();
+
+    match client.list_certificates(&request) {
+    	Ok(response) => {
+    		println!("{:#?}", response); 
+    		assert!(true)
+    	},
+    	Err(err) => panic!("Expected OK response, got {:#?}", err)
+    };
```

Comment:
I think all these tests could be re-written as:

```
client.list_certificates(&request).unwrap();
```

Because:

1. The `println!`s don't do anything in the tests
2. `assert!(true)` is weird
3. If the result is an error and we unwrap, the panic will print the error description anyways.